viz_scripts/bin/generate_plots.py

```diff
@@ -34,13 +34,34 @@
     dynamic_config = json.loads(r.text)
     print(f"Successfully downloaded config with version {dynamic_config['version']} "\
         f"for {dynamic_config['intro']['translated_text']['en']['deployment_name']} "\
-        f"and data collection URL {dynamic_config['server']['connectUrl']}")
+        f"and data collection URL {dynamic_config['server']['connectUrl'] if 'server' in dynamic_config else 'default'}")
 
 if dynamic_config['intro']['program_or_study'] == 'program':
     mode_studied = dynamic_config['intro']['mode_studied']
 else:
     mode_studied = None
 
+# dynamic_labels can  be referenced from 
+# https://github.com/e-mission/nrel-openpath-deploy-configs/blob/main/label_options/example-study-label-options.json
+dynamic_labels = { }
+
+# Check if the dynamic config contains dynamic labels 'label_options'
+# Parse through the dynamic_labels_url:
+if 'label_options' in dynamic_config:
+    dynamic_labels_url = dynamic_config['label_options']
+
+    if  dynamic_labels_url:
+        req = requests.get(dynamic_labels_url)
+        if req.status_code != 200:
+            print(f"Unable to download dynamic_labels, status code: {req.status_code}")
+        else:
+            dynamic_labels = json.loads(req.text)
+            print("Dynamic labels download was successful.")
+    else:
+        print("Dynamic labels URL is unavailable.")
+else:
+    print("Dynamic labels are not available.")
+
```

Comment:
nit: (future cleanup): make the log statements more meaningful.

`is unavailable` is essentially the same as `are not available` it would be better to have more meaningful log statements.

Reply:
Updated the log messages.